app/components/Views/AccountConnect/AccountConnect.tsx

```diff
@@ -169,28 +171,27 @@ const AccountConnect = (props: AccountConnectProps) => {
   const isOriginWalletConnect =
     !isOriginMMSDKRemoteConn && wc2Metadata?.id && wc2Metadata?.id.length > 0;
 
-  const { isEip1193Request } = hostInfo.metadata;
-
-  const defaultSelectedChainIds = useMemo(() => {
-    // For EIP-1193 requests (injected Ethereum provider requests) or WalletConnect or MMSDK Remote Conn,
-    // we only want to show EIP-155 (Ethereum) compatible chains
-    if (isEip1193Request || isOriginWalletConnect || isOriginMMSDKRemoteConn) {
-      return allNetworksList.filter((chain) =>
-        chain.includes(KnownCaipNamespace.Eip155),
-      );
-      // otherwise, if we have supported requested CAIP chain IDs, use those
-    } else if (supportedRequestedCaipChainIds.length > 0) {
-      return supportedRequestedCaipChainIds;
-    }
-    // otherwise, use all available networks
-    return allNetworksList;
-  }, [
-    isEip1193Request,
-    allNetworksList,
-    isOriginWalletConnect,
-    isOriginMMSDKRemoteConn,
-    supportedRequestedCaipChainIds,
-  ]);
+  const defaultSelectedChainIds = useMemo(
+    () =>
+      getDefaultSelectedChainIds({
+        isEip1193Request: !!isEip1193Request,
+        isOriginWalletConnect: !!isOriginWalletConnect,
+        isOriginMMSDKRemoteConn: !!isOriginMMSDKRemoteConn,
```

Comment:
```suggestion
        isEip1193Request: Boolean(isEip1193Request),
        isOriginWalletConnect: Boolean(isOriginWalletConnect),
        isOriginMMSDKRemoteConn: Boolean(isOriginMMSDKRemoteConn),
```

nit. I see more usages of `Boolean()` than `!!` in the repo